templates/template-app/main/CMakeLists.txt

```diff
@@ -5,4 +5,4 @@ set(COMPONENT_PRIV_REQUIRES )
 set(COMPONENT_SRCS "main.c")
 set(COMPONENT_ADD_INCLUDEDIRS "")
 
-register_component()
\ No newline at end of file
+idf_component_register()
```

Comment:
The list of source and include directories should be passed to idf_component_register. You can refer to https://docs.espressif.com/projects/esp-idf/en/latest/api-guides/build-system.html#minimal-component-cmakelists

Reply:
Added in #38